src/benchmark/benchmark.cc

```diff
@@ -25,6 +25,8 @@
   Program grant you additional permission to convey the resulting work.
 */
 
+#include <numeric>
```

Comment:
I understand the style used requires this after `#include "benchmark/benchmark.h"` but before `#include "mcts/search.h"`

Reply:
Should be addressed by commit 2257352 now.